docs/tools/firmware.sh

```diff
@@ -59,16 +201,23 @@ case "$os" in
 			echo "Exiting..."
 			exit 1
 		fi
-
-		echo "Re-mounting the EFI partition"
+		echo -e "\nIf you are running this script in Linux, make sure you ran it first in macOS and choose option 1 (Run the same script on Linux)."
+		echo -e "\nContinue? (Y/n)"
+		read input
+		if [[ ($input = n) || ($input = N) ]]
+		then
+			echo -e "\nRun the script again in Linux when you have ran it in macOS."
+			exit 1
+		fi
+		echo -e "\nRe-mounting the EFI partition"
 		mountpoint=$(mktemp -d)
 		workdir=$(mktemp -d)
 		echo "Installing Wi-Fi and Bluetooth firmware"
 		sudo mount ${verbose} /dev/nvme0n1p1 $mountpoint
 		sudo tar --warning=no-unknown-keyword ${verbose} -xC $workdir -f $mountpoint/firmware.tar.gz
-		sudo python3 $0 $workdir $workdir/firmware-renamed.tar ${verbose}
+		sudo python3 "$0" $workdir $workdir/firmware-renamed.tar ${verbose}
 
-		sudo tar ${verbose} -xC /lib/firmware -f $workdir/firmware-renamed.tar
+		sudo tar ${verbose} -xC /lib/firmware/brcm -f $workdir/firmware-renamed.tar
```

Comment:
Can you revert this (here and in the python script), I think its better if the tarball is the contents of `/usr/lib/firmware` so we can add other dirs into there if we need to.

Reply:
I did the change so that users who choose option 2 don't get confused. I wouldn't be surprised if I get people copying the brcm folder in brcm of linux.

Reply:
I guess its fine then